coalib/results/Diff.py

```diff
@@ -285,10 +317,11 @@ def unified_diff(self):
         Note that the unified diff is not deterministic and thus not suitable
         for equality comparison.
         """
-        return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+        tofile = (self.rename if isinstance(self.rename, str) else
+                  self.create if isinstance(self.create, str) else '')
+
+        return ''.join(difflib.unified_diff(self.original, self.modified,
```

Comment:
Formatting looked a little off to me and it could be improved with current change, so changed it.. But sure will change it